modules/tinymce/src/core/test/ts/browser/lookup/UserLookupTest.ts

```diff
@@ -11,18 +11,25 @@ import { createUserLookup, type User } from 'tinymce/core/lookup/UserLookup';
 
 Chai.use(chaiAsPromised);
 
+interface ExpectedUser {
+  id: string;
+  [key: string]: any;
+};
+
 const createMockUser = (id: string): User => ({
   id,
   name: 'Test User',
   avatar: 'test-avatar.png',
-  description: 'Test Description'
+  custom: {
+    description: 'Test Description'
+  }
 });
 
 describe('browser.tinymce.core.UserLookupTest', () => {
   const hook = TinyHooks.bddSetupLight<Editor>({
     base_url: '/project/tinymce/js/tinymce',
     user_id: 'test-user-1',
-    fetch_users: (userIds: string[]): Promise<User[]> => {
+    fetch_users: (userIds: string[]): Promise<ExpectedUser[]> => {
```

Comment:
If it used here as a type, wouldn't we type this in the public interface too with the same time and therefor not need the local type definition in this file.
Also I think we should be able to put that in the type for fetch_users in the options right so in OptionTypes? Then I think you might not need to annotate this function at all an the implementors of this API would be guided.